db/migrate/20170508150135_add_foreign_keys.rb

```diff
@@ -0,0 +1,31 @@
+class AddForeignKeys < ActiveRecord::Migration
+  def change
+    add_index :notifications, [:subject_id, :subject_type]
+    add_index :order_details, :project_id
+    add_index :order_statuses, :parent_id
+    add_index :orders, :merge_with_order_id
```

Comment:
Is there a way we can make the `add_index` conditional on the index not already being there? Or maybe have it `rescue` and move on?